### DISCUSSION TITLE
[Feature] Grafana alerting should support variables

### DISCUSSION THREAD

**Discussion by aaawuanjun** [grafana/grafana#56921] [Alerting]
# Example

when we should monitor multi machine， we record every machine node in Variables.
but we we want to alter to us, it noticed "Template variables are not supported in alert queries"

![image](https://user-images.githubusercontent.com/35732071/195792878-9526de00-a770-41c3-abfe-628e41be2ed7.png)

we do not want to expand the variables， when we have many machine node, it is not unfriendly.
![image](https://user-images.githubusercontent.com/35732071/195793240-2db3cdb7-1783-400b-a384-e186a8b5e61f.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
